src/pybind/mgr/dashboard/frontend/src/app/ceph/pool/pool-form/pool-form.component.html

```diff
@@ -427,6 +427,58 @@
             </div>
           </div>
 
+        <!-- Quotas -->
+        <div>
+          <legend i18n>Quotas</legend>
+
+          <!-- Max Bytes -->
+          <div class="form-group"
+               [ngClass]="{'has-error': form.showError('max_bytes', formDir)}">
+            <label class="control-label col-sm-3"
+                   for="max_bytes">
+              <ng-container i18n>Max bytes</ng-container>
+            </label>
+            <div class="col-sm-9">
+              <input class="form-control"
+                     id="max_bytes"
+                     name="max_bytes"
+                     type="text"
+                     formControlName="max_bytes"
+                     i18n-placeholder
+                     placeholder="e.g., 10GiB"
+                     defaultUnit="GiB"
+                     cdDimlessBinary>
+              <span class="help-block"
+                    i18n>Specify 0 to disable quota</span>
+              <span class="help-block"
+                    *ngIf="form.showError('max_bytes', formDir, 'required')"
```

Comment:
Think this should reflect the `min` validation or both
```suggestion
                    *ngIf="form.showError('max_bytes', formDir, 'min')"
```

Reply:
removed this error condition since min validator does not apply on text input